tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -158,20 +157,48 @@ public static function prepare($scope)
     /**
      * This hook can be used to flag a feature for database hard reset
      *
+     * @deprecated since 8.0.0 and will be removed in next major.
+     *
      * @BeforeFeature @reset-database-before-feature
      */
     public static function cleanDatabaseHardPrepareFeature()
     {
+        @trigger_error(
```

Comment:
Method can be removed, it's related to tests

Reply:
I prefer to keep it until all the test have been optimized for now We can remove it once we're sure it's not useful at all any more